js-api-doc/importer.d.ts

```diff
@@ -308,6 +308,37 @@ export interface Importer<sync extends 'sync' | 'async' = 'sync' | 'async'> {
   nonCanonicalScheme?: string | string[];
 }
 
+/**
+ * An opaque branded type, representing the built-in Node Package Importer. This
+ * is used for type checking. Users should use the exposed `nodePackageImporter`
```

Comment:
Use `{@link nodePackageImporter}` to link to the member rather than just referring to it by name.